tests/data/ref_dim_keyword.nxdl.xml

```diff
@@ -30,7 +30,7 @@
             <doc>
```

Comment:
Can we add a test for the other direction, i.e., if we have <dimensions> in the xml, how does this translate to nyaml?

Reply:
Acctullay, I did not noticed no dimension test from nxdl to yaml conversion. 
I will add it.

Reply:
I added a proper test script for nyaml2nxdl dimension and dim keyword.

Reply:
Great thanks!